### PR TITLE
Improve issue template appealingness

### PR DESCRIPTION
**Pull request by zoontek**
# Summary

Following a discussion with @brentvatne, we agreed on would be nice to improve a bit the default issue template (remind to check for duplicates, compatibility).

Often, I see the templates not respected, deleted immediately. It makes everyone lose time (the lib user and the maintainers).

I know it should be common practice to just close the issues in question, but IMHO it's not ideal and it might seem harsh to newcomers. So I tried making the bug report a bit more appealing to _maybe_ improve the conversation rate.

PS: Note that English is not my primary language 😅

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,53 +1,66 @@
...
 
-  **Please note that issues that do not follow the template may be closed.**
+🚨 Please read the following carefully before opening a new issue. Your issue may
+be closed if it doesn't provide the required pieces of information. 🚨
+
+Before submitting a new issue, please:
+
+- Test using the latest release of the library, as maybe your bug has been already fixed.
+- Check for possible duplicate issues, with possible answers.
+- Check on https://reactnative.directory if this package supports your targetted platform (Android, iOS, Expo, …)
```
Comment (brentvatne) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
- Check on https://reactnative.directory if this package supports your target platform (Android, iOS, Expo, …)
```

**Review by brentvatne — APPROVED** [review]
this looks good! hopefully it leads to some higher quality issue reports!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (zoontek) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Updated, thanks!

**Comment by kelset** [comment]
Let's goooo thanks for working on this, great idea 👍

**Pull request merged**